Change example Job Admission Policy to use matchLabels (#2601)
kubernetes-sigs/kueue: site/content/en/docs/tasks/manage/setup_job_admission_policy.md

kubectl create -f sample-validating-policy-binding.yaml
```

> Label each namespace where you want this policy to be enforced by doing:

Reviewer: ```suggestion
Run the following command to label each namespace where you want this policy to be enforced:
```
